axelar-chains-config/info/mainnet.json

```diff
@@ -122,7 +122,8 @@
       "staticGasOptions": {
         "gasLimit": 3000000,
         "gasPrice": 150000000000
-      }
+      },
+      "finality": "finalized"
```

Comment:
update the schema in the axelar-chains-config folder, and update the `info.test.js` to test this setting